docs/guides/latest-updates.mdx

```diff
@@ -29,6 +29,23 @@ Jump to [Qiskit SDK](#latest-qiskit-sdk) | [Qiskit Runtime client](#latest-qiski
 
 This section summarizes the recent enhancements and new features for IBM Quantum Platform.
 
+**24 April 2025**
+
+- Instance limits: Individuals can view [available time and usage](https://quantum.cloud.ibm.com/instances), which displays alerts when approaching account or instance limits.
+- Admin permissions: Admins can view overall usage, cycle period, and remaining time per plan in the instances page.
```

Comment:
```suggestion
- Administrator permissions: Administrators can view overall usage, cycle period, and remaining time per plan in the Instances page.
```

Reply:
Does "overall usage" mean "usage for that instance"?  Can they also see how much is remaining on an instance's allocation?  Maybe there should be an "and more" at the end of the above statement?

Reply:
@lerongil clarity question here ⬆️

Reply:
This can be replaced with the following for clarity: 

Administrator Permissions: Administrators can view the available usage, remaining usage, and cycle period for each plan on the Instances page.